cg_hermes/config/nextflow.py

```diff
@@ -4,12 +4,24 @@
 mandatory by default. However the tags that are available to a particular analysis is mandatory for that analysis.
 """
 
+from cg_hermes.constants.tags import (
+    NextflowTags,
+    QCTags,
+    UsageTags,
+)
+
+
 NEXTFLOW_TAGS = {
     frozenset({"software-versions"}): {
         "is_mandatory": True,
         "bundle_id": True,
-        "tags": ["software-versions"],
-        "used_by": ["cg", "long-term-storage"],
+        "tags": [NextflowTags.SOFTWARE_VERSIONS],
+        "used_by": [UsageTags.CG, UsageTags.LONG_TERM_STORAGE],
+    },
+    frozenset({"qc-metrics"}): {
+        "is_mandatory": True,
+        "tags": [QCTags.QC_METRICS],
+        "used_by": [UsageTags.CG, UsageTags.LONG_TERM_STORAGE],
     },
```

Comment:
We need to remove it from the taxprofiler config too

Reply:
Done